src/kernels/jupyter/session/jupyterRequestCreator.web.ts

```diff
@@ -43,20 +43,15 @@ export class JupyterRequestCreator implements IJupyterRequestCreator {
                 if (allowUnauthorized) {
                     // rejectUnauthorized not allowed in web so we can't do anything here.
                 }
-
-                if (cookieString) {
-                    this.headers.append('Cookie', cookieString);
```

Comment:
Just pass in `getAuthHeaders`, hence no more custom handling for user provided servers and cookies.